benchmarks/benchmarks.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2024 D-Wave Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import subprocess
+
+from functools import partial
+from pathlib import Path
+
+from dwave.cloud import Client
+from dwave.cloud.coders import encode_problem_as_qp
+from dwave.cloud.config import ClientConfig
+from dwave.cloud.solver import StructuredSolver, BQMSolver, CQMSolver, DQMSolver, NLSolver
+from dwave.cloud.regions import resolve_endpoints, get_regions
+from dwave.cloud.testing import isolated_environ, mocks
+from dwave.cloud.utils.qubo import generate_random_ising_problem
+
+
+# note: looks like timeraw benchmarks can't be parameterized
+class ImportDeps:
+    version = "1"
```

Comment:
Is there a need to add version to all the classes as well as in the conf file, or is this useful for some other reason?

Reply:
This is individual benchmark version (see `version` in [docs](https://asv.readthedocs.io/en/v0.6.3/benchmarks.html#benchmark-attributes)). I'm specifying it explicitly to be able to control it better. Otherwise a commit hash is used, and when a minor code change happens, previous results are dropped.